dotCMS/src/main/java/com/dotmarketing/cms/webforms/action/SubmitWebFormAction.java

```diff
@@ -55,7 +56,7 @@
  * @author David
  * @version $Revision: 1.5 $ $Date: 2007/07/18 16:48:42 $
  */
-public final class SubmitWebFormAction extends DispatchAction {
+public final class SubmitWebFormAction extends SecureAction {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SecureAction"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)